### PR TITLE
Yee no longer unmutates mutants

### PR DESCRIPTION
**Pull request by Superlagg**
[FEATURE]

## About the PR 

If a mutant race spaceperson became afflicted with yee, once it wore off they'd revert to being a boring old human.

It doesn't do that anymore. Your mutant race is preserved!

Also, yee now penetrates skin, since that was the only sane way I could find to make this work with admeme yeefloods. I mean, it already reacted on touch, so it shouldn't really change anything.

## Why's this needed? 

As a career lizard, I want to appreciate the horrors of yee and still remain a lizard when it runs itself into the ground. Also, technically it could be used to shut down werewolves. I mean, it still kinda can, but they go back to being a werewolf once the yee wears out.

### CODE REVIEW

File: code/modules/chemistry/Reagents-Misc.dm [hunk 1]
```diff
@@ -3353,31 +3353,43 @@ datum
 			fluid_g = 97
 			fluid_b = 10
 			transparency = 225
+			penetrates_skin = 1
 			var/music_given_to = null
+			var/furry = null
+			var/furrie = null
```
Comment (Kyle2143) [inline code/modules/chemistry/Reagents-Misc.dm:3359]: These variable names aren't particularly descriptive...

Comment (ZeWaka) [inline code/modules/chemistry/Reagents-Misc.dm:3359]: rename these to actually descriptive vars please.

Comment (Superlagg) [reply code/modules/chemistry/Reagents-Misc.dm]: Done.

Comment (Superlagg) [reply code/modules/chemistry/Reagents-Misc.dm]: And done.

File: code/modules/chemistry/Reagents-Misc.dm [hunk 2]
```diff
@@ -3353,31 +3353,43 @@ datum
 			fluid_g = 97
 			fluid_b = 10
 			transparency = 225
+			penetrates_skin = 1
 			var/music_given_to = null
+			var/stored_furry_bioeffect = null
+			var/stored_furry_mutantrace = null
```
Comment (ZeWaka) [inline code/modules/chemistry/Reagents-Misc.dm:3359]: Still not aptly named.

Comment (Superlagg) [reply code/modules/chemistry/Reagents-Misc.dm]: Done.

**Comment by Gerhazo** [comment]
Probably doesn't need a changelog entry with how minor and out of the way this is.
At least definitely not a major one.

**Pull request merged**